src/plugins/discover/public/application/apps/main/components/sidebar/discover_field_details.tsx

```diff
@@ -84,30 +41,35 @@ export function DiscoverFieldDetails({
               />
             ))}
           </div>
-        )}
-
-        {showVisualizeLink && (
-          <>
-            <EuiSpacer size="xs" />
-            {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
-            <EuiButton
-              onClick={(e) => handleVisualizeLinkClick(e)}
-              href={visualizeLink}
-              size="s"
-              className="dscFieldDetails__visualizeBtn"
-              data-test-subj={`fieldVisualize-${field.name}`}
-            >
+          <EuiSpacer size="xs" />
+          <EuiText size="xs">
+            {!indexPattern.metaFields.includes(field.name) && !field.scripted ? (
+              <EuiLink
+                onClick={() => onAddFilter('_exists_', field.name, '+')}
+                data-test-subj="onAddFilterButton"
+              >
+                <FormattedMessage
+                  id="discover.fieldChooser.detailViews.existsInRecordsText"
+                  defaultMessage="Exists in {value} / {totalValue} records"
+                  values={{
+                    value: details.exists,
+                    totalValue: details.total,
+                  }}
+                />
+              </EuiLink>
+            ) : (
               <FormattedMessage
-                id="discover.fieldChooser.detailViews.visualizeLinkText"
-                defaultMessage="Visualize"
+                id="discover.fieldChooser.detailViews.valueOfRecordsText"
+                defaultMessage="{value} / {totalValue} records"
+                values={{
+                  value: details.exists,
+                  totalValue: details.total,
+                }}
               />
-            </EuiButton>
-            {warnings.length > 0 && (
-              <EuiIconTip type="alert" color="warning" content={warnings.join(' ')} />
             )}
```

Comment:
:information_source: This is the mentioned scripted field performance warning that's removed (it was never possible to show any other warning than that one).